azureVMCreateCommentVariables.md

```diff
@@ -0,0 +1,159 @@
+---
```

Comment:
Shouldn't be in the root folder and it looks like a parameterized version of azureVMCreate.md - we should only have this version.